content/operate/rc/databases/create-database/create-flex-database.md

```diff
@@ -0,0 +1,140 @@
+---
+Title: Create a Redis Flex database
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: Shows how to create an Redis Flex database on Redis Cloud and describes the best use cases for Redis Flex. 
+linkTitle: Create Redis Flex database
+weight: 8
+tocEmbedHeaders: true
+---
+
+Redis Flex databases have a combined solid state drive (SSD) and RAM architecture. Using SSDs instead of RAM significantly reduces infrastructure costs, which means developers can build applications that require large datasets using the same Redis API.
```

Comment:
```suggestion
Redis Flex databases have a tiered solid state drive (SSD) and RAM architecture. Using SSDs instead of RAM significantly reduces infrastructure costs, which means developers can build applications that require large datasets using the same Redis API.
```